api/tasks/swagger-api_tasks.yaml

```diff
@@ -0,0 +1,23 @@
+openapi: 3.0.1
+info:
+  title: Swagger Cloudtrust Management
+  description: 'Tasks API for Cloudtrust.'
+  version: 1.0.0
+servers:
+- url: http://localhost:8888
+tags:
+- name: Tasks
+  description: Tasks management
+paths:
+  /tasks/denied-terms-of-use-users:
+    delete:
+      tags:
+      - Tpackage pkgtasks
+asks
```

Comment:
I think this word should not be here ! It brokes the swagger syntax

Reply:
Good catch ! :-)
That's what happens when you have multiple displays, you type CTRL+C but you are not watching the correct display...